### PR TITLE
Fbb endpoints

### PR DESCRIPTION
**Pull request by devang-dreambits**

### CODE REVIEW

File: bol/plaza/models.py [hunk 1]
```diff
@@ -377,7 +377,6 @@ class Meta:
         AnnouncedQuantity = IntegerField()
         ReceivedBSKUs = IntegerField()
         ReceivedQuantity = IntegerField()
-        EAN = IntegerField()
```
Comment (karan-dreambits) [inline bol/plaza/models.py]: Is this removed by mistake or was added before by mistake ?

File: bol/plaza/api.py [hunk 2]
```diff
@@ -503,8 +576,39 @@ def getDeliveryWindow(self, delivery_date=None, items_to_send=None):
 
         response = self.request_inbound('GET', path='delivery-windows',
                                         params=params)
+
```
Comment (karan-dreambits) [inline bol/plaza/api.py]: Why this extra space ?

File: tests/test_plaza.py [hunk 3]
```diff
@@ -650,7 +650,8 @@ def delivery_window_stub(url, request):
     with HTTMock(delivery_window_stub):
         api = PlazaAPI('api_key', 'api_secret', test=True)
 
-        param_date = datetime.strptime('30-01-2017', '%d-%m-%Y').date()
+        # param_date = datetime.strptime('30-01-2017', '%d-%m-%Y').date()
```
Comment (karan-dreambits) [inline tests/test_plaza.py]: Please remove this comment

File: bol/plaza/models.py (hunk 1 above)
Comment (devang-dreambits) [reply bol/plaza/models.py]: There is no case it should be there ... it was added before by mistake

File: bol/plaza/api.py (hunk 2 above)
Comment (devang-dreambits) [reply bol/plaza/api.py]: pep-8 check says that there should be a blank line at the end of the file, ...

even git diff also says the same

git diff
diff --git a/bol/plaza/api.py b/bol/plaza/api.py
index af05594..4541f29 100644
--- a/bol/plaza/api.py
+++ b/bol/plaza/api.py
@@ -722,4 +722,4 @@ x-bol-date:{date}
                 return tree
         except Exception:
             print("Got into Exception \n{0}".format(traceback.print_exc()))
-            return False
+            return False
\ No newline at end of file

Comment (karan-dreambits) [reply bol/plaza/api.py]: ohh yes. My mistake. Didn't see that its end of file :man_facepalming:

Comment (karan-dreambits) [reply bol/plaza/api.py]: But this is not the end of file. There are 200+ more lines below this

File: bol/plaza/models.py (hunk 1 above)
Comment (devang-dreambits) [reply bol/plaza/models.py]: There is no case it should be there ... it was added before by mistake

File: tests/test_plaza.py (hunk 3 above)
Comment (devang-dreambits) [reply tests/test_plaza.py]: removed ...

File: bol/plaza/api.py (hunk 2 above)
Comment (karan-dreambits) [reply bol/plaza/api.py]: @devang-dreambits But this is not end of file, Is it ?

**Pull request merged**